slyd/slyd/api.py

```diff
@@ -77,7 +77,10 @@ def render(self, request):
                                   for k, v in parsed.named.items()}
                         parsed['attributes'] = load_attributes(request)
 
-                        api_response = callback(manager, **parsed)
+                        api_response = getattr(
```

Comment:
lets make run defined on the base manager classes as a noop and explicitly call it here